v1.2/armhf/debian-onbuild/Dockerfile

```diff
@@ -1,9 +1,9 @@
 # AUTOMATICALLY GENERATED
 # DO NOT EDIT THIS FILE DIRECTLY, USE /Dockerfile.template.erb
 
-FROM resin/armv7hf-debian:stretch
+FROM arm64v8/debian:stretch-slim
```

Comment:
ARMv7 is not ARMv8 platform.
They are different architecture.
ARMv8 platform can execute ARMv7 binary as ARM 32bit binary but  ARMv7 platform cannot execute ARMv8 binary.
So, if we would support this platform, you should add ARMv8 Dockerfiles instead of just replace ARMv7 Dockerfiles.